documentation/source/quickstart.rst

```diff
@@ -230,12 +231,17 @@ Values can be assigned to signals using either the .value property of a handle o
     clk.value = 1
     
     # Direct assignment through the hierarchy
-    dut.input_signal = 12
+    dut.input_signal <= 12 
 
     # Assign a value to a memory deep in the hierarchy
-    dut.sub_block.memory.array[4] = 2
-        
-        
+    dut.sub_block.memory.array[4] <= 2
+
+
+The "<=" operator is overridden by cocotb to help make it more clear when an object
+being assigned a value is part of the RTL source as compared to the python test code. 
```

Comment:
I don't think this is the real reason. I think this syntax is chosen because:

* It emulates non-blocking assignment - the assignment doesn't happen until the next write cycle
* It allows:
    ```
    some_sig = dut.input_signal
    some_sig <= 1
    # some_sig = 1 would just rebind the name
    ```

Reply:
``some_sig = dut.input_signal`` could be seen as the equivalent to a "signal map" in UVM-e. (There is probably the same thing with a different name in UVM-SV.)
It would be nice if we could create a cocotb environment that is familiar to UVM users, and documenting such equivalences are a start.

Reply:
We'd need Stu or Chris to chime in about 'real' reasons. I'm up for clarifying this, but from the comments above I'm not sure how to go about that.